src/current/_includes/v24.3/cdc/msk-iam-policy-role-step.md

```diff
@@ -0,0 +1,51 @@
+1. In the AWS Management Console, go to the [IAM console](https://console.aws.amazon.com/iam/), select **Policies** from the navigation, and then **Create Policy**.
+1. Using the **JSON** tab option, update the policy with the following JSON. These permissions will allow you to connect to the cluster, manage topics, and consume messages. You may want to adjust the permissions to suit your permission model. For more details on the available permissions, refer to the AWS documentation on [IAM Access Control](https://docs.aws.amazon.com/msk/latest/developerguide/iam-access-control.html#kafka-actions) for MSK.
+
+    Replace the instances of `arn:aws:kafka:{region}:{account ID}:cluster/{msk-cluster-name}` with the MSK ARN from your cluster's summary page.
```

Comment:
There's a slight mismatch in these text strings - the string in the JSON ends with a `/*` 

oh, i get it - replace the substring - maybe it's worth saying "don't delete the trailing `*` " or similar for folks not reading carefully

Reply:
(assuming i understood this correctly)